venues/OpenReview.net/Support/deployProcess.py

```diff
@@ -176,16 +176,6 @@ def process(client, note, invitation):
                 'required': True,
                 'order': 11
             },
-            'make_reviews_public': {
```

Comment:
can we call it 'release_to_public'?

In fact, I think we should merge 'release_reviews_to_authors' too? Like a radio button that says:

```
Release public to:

- Public
- Reviewers and Authors only
- Submitted reviewers only
```

Reply:
The default option is submitted reviewers only I guess

Reply:
For the second choice, it is all reviewers for that paper and authors? I tend to think it should be submitted reviewers and authors.
Also this means we should change the "Open Reviewing Policy" question on setup to be just about submissions, not submissions and reviews.

Reply:
Let's do the mapping of choices with the Review Stage parameters:

- Public (public = True)
- All Paper Reviewers (public = False, release_to_authors = False, release_to_reviewers = True)
- All Paper assigned Reviewers and Authors (public = False, release_to_authors = True, release_to_reviewers = True)
- Only submitted Paper assigned Reviewers (public = False, release_to_authors = False, release_to_reviewers = False) (default option)